users/tests/test_models.py

```diff
@@ -46,7 +46,7 @@ def test_exception_permissions(self):
                 user_instructeur.has_perm(perm, user_instructeur)
                 self.fail(
                     f"has_perm '{perm}' "
-                    + "with non convention object shold raise an Exception"
+                    + "with non convention object should raise an Exception"
```

Comment:
```suggestion
                    "with non convention object should raise an Exception"
```

En principe en python on peut écrire une string sur plusieurs lignes, donc pas besoin du `+` ici